src/app/[locale]/home/components/MyComponentsWidget.tsx

```diff
@@ -24,7 +26,14 @@ function MyComponentsWidget() {
     useEffect(() => {
         const fetchData = async () => {
             const data = await sw360FetchData('/components/mycomponents', 'components')
-            setData(data.map((item: { name: string, description: string }) => [item.name, item.description]))
+            if (data.length > 0)
```

Comment:
You can write this code snippets like this. Same with other files
```
if (data.length > 0) {
data = defaultMyComponents
}
setData(data.map((item: { name: string, description: string }) => [item.name, item.description]))
```

Reply:
Should we have defaultValues ? Should not be the string for empty ?
Example:
https://gridjs.io/docs/config/className -> Look for notFound

And gridjs code already has a default message for empty:
```
      {status === Status.Rendered && data && data.length === 0 && (
        <MessageRow
          message={_('noRecordsFound')}
          colSpan={headerLength()}
          className={classJoin(
            className('notfound'),
            config.className.notfound,
          )}
        />
      )}
```

Reply:
This also looks fine. But I was thinking of showing table specific messages

For example : 

For `My Projects` table : We can show, `No projects found` , or 
For `My Components` table : We can show, `No components found` 

What you people think about it ?

Reply:
@amritkv. In my opinion, no record found message is enough